scripts/promote-config.sh

```diff
@@ -36,6 +36,10 @@ main() {
     # except for manifest.yaml
     git checkout -- manifest.yaml
 
+    # also strip out the snoozes in the denylist because we don't want
+    # changes in the executed tests over time for production streams
+    sed -E -i '/^\s+snooze/d' kola-denylist.yaml
```

Comment:
How about

```suggestion
    sed -E -i 's/^(\s+)(snooze:\s+.*)/\1# \2 (disabled on promotion)/' kola-denylist.yaml
```

?

That way there's a paper trail of what this script did.

Reply:
updated!